docs/design.md

```diff
@@ -186,22 +187,23 @@ timestamps to diverge from wall clock time, following closely the
 [*Hybrid Logical Clock
 paper.*](http://www.cse.buffalo.edu/tech-reports/2014-04.pdf)
 
-Transactions are executed in two phases:
+Transactions are executed in discrete phases:
```

Comment:
Maybe reorg this section slightly so it really is two phases. So the first phase would include steps 1 & 2 (maybe change them to bullets). And the second phase would be step 3.

The number 2 is important here. Just changing this to "discrete phases" misses the point.